pyproject.toml

```diff
@@ -77,6 +76,26 @@ Tidelift = "https://tidelift.com/subscription/pkg/pypi-jsonschema?utm_source=pyp
 Changelog = "https://github.com/python-jsonschema/jsonschema/blob/main/CHANGELOG.rst"
 Source = "https://github.com/python-jsonschema/jsonschema"
 
+[tool.hatch.metadata.hooks.fancy-pypi-readme]
+content-type = "text/x-rst"
+
+[[tool.hatch.metadata.hooks.fancy-pypi-readme.fragments]]
+path = "README.rst"
+
+[[tool.hatch.metadata.hooks.fancy-pypi-readme.fragments]]
+text = """
+
+
+===================
+Release Information
+===================
```

Comment:
I think for the same reason as the comment this should be just under-`-` unless we do change the headings in the `README`, otherwise it'll turn into an H1 (and be the only other H1), no?

Reply:
yes, but your changelog consists of `-`-headings which means they'd become equal to your sections.

Making it `=` would \~probably\~ make it h2 and all your sections h3.

Reply:
I think if we want to make believe this problem doesn't exist :D we can just change all the headings in the Changelog to be single `=` and then change this to be single `-` (which will make the heading H2, and then each changelog entry be an H3)